_pages/en_US/installing-boot9strap-(safecerthax).txt

```diff
@@ -29,7 +29,12 @@ This exploit will not work on the New 3DS, New 3DS XL, or New 2DS XL. Please ens
 
 #### Section I - Hardware Button Check
 
-{% include_relative include/safemodecheck.txt %}
+In this section, you will see whether your shoulder buttons are working on your console. This will determine which method you will follow on the next page.
```

Comment:
hmm, this isn't correct in safecerthax

apparently we were A-OK with that this whole time lmao